RELEASE_NOTES.md

```diff
@@ -1,3 +1,15 @@
+# ChefDK 3.10 Release Notes
+
+## New Policy File Functionality
+
+`include_policy` now supports `:remote` policy files. This allows you to include policy files over http. Remote policy files require remote cookbooks and `install` will fail if the included policy file includes cookbooks with paths. Thanks @mattray!
```

Comment:
```suggestion
`include_policy` now supports `:remote` policy files. This new functionality allows you to include policy files over http. Remote policy files require remote cookbooks and `install` will fail otherwise if the included policy file includes cookbooks with paths. Thanks @mattray!
```